### DISCUSSION TITLE
helix as an html/css/ruby editor

### DISCUSSION THREAD

**Discussion by work-leonid** [helix-editor/helix#2054] [Q&A] [answered]
Hi all, I just started switching to neovim and came across helix. And I really liked it. It's great that a lot of things work out of the box. I would like to use it as an everyday editor for html, css and ruby/rails. 

Right now I'm missing a few features
- emmet for html/css. Maybe it's there or is it planned?
- for some reason the highlighting doesn't work for html.erb files, but it works for rb
- maybe there are articles/videos on setting up helix for rails. I tried to go by the documentation, but it's too obscure for a newbie right now

But maybe i helix is not suitable for my purposes?

**Comment by sudormrfbin** [comment, marked as answer]
Previous emmet discussion: https://github.com/helix-editor/helix/issues/1896. And an introductory video on helix: https://youtu.be/9Zj-wiQ9_Xw.

Edit: For erb there's an open PR - https://github.com/helix-editor/helix/pull/2055.

**Reply by the-mikedavis** [reply]
Also for ERB, I just opened a PR #2055 :)

**Reply by work-leonid** [reply]
Thank you, @sudormrfbin 
- Support for emmet will be in future releases. But there is no exact date.
- erb.html looks like it will be done faster

I think this https://youtu.be/9Zj-wiQ9_Xw video is what attracts all newcomers. Too bad it's the only one.  I can't even find articles about helix.

**Reply by work-leonid** [reply]
And we can add auto pairs for <> brackets https://docs.helix-editor.com/configuration.html#editorauto-pairs-section which will simplify working with html

**Reply by the-mikedavis** [reply]
For that, we could add a section like [this](https://github.com/helix-editor/helix/blob/562874a720678a238966c69f413cb3d691f74c66/languages.toml#L12-L17) to right [here](https://github.com/helix-editor/helix/blob/562874a720678a238966c69f413cb3d691f74c66/languages.toml#L359). Would you like to make a pull request that adds `<`-`>` auto-pairs for html?

**Reply by work-leonid** [reply]
> For that, we could add a section like [this](https://github.com/helix-editor/helix/blob/562874a720678a238966c69f413cb3d691f74c66/languages.toml#L12-L17) to right [here](https://github.com/helix-editor/helix/blob/562874a720678a238966c69f413cb3d691f74c66/languages.toml#L359). Would you like to make a pull request that adds `<`-`>` auto-pairs for html?

Yes, I'll give it a try.

**Reply by pickfire** [reply]
I use it for django, it works like a plain text editor but haven't got the time to do django template tree-sitter for it yet.
